tests/base_scraper/test_correct_scraper.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import unittest
+import urllib
+
+import mozfile
+import mozhttpd_base_test as mhttpd
+from mozprocess import processhandler
+
+tests = [
+    # ReleaseScraper
+    {'command': ['-v', 'latest'],
+     'fname': 'firefox-latest.en-US.linux64.tar.bz2'},
+    # ReleaseCandidateScraper
+    {'command': ['-t', 'candidate', '-v',
+                 '21.0', '-p', 'win32',],
```

Comment:
When you break lines, please do in such a way that we do not separate options which have to be used together.